Added robustness against corrupt ProjectVersion.txt files & added resizeable columns

geo-at-github opened the pull request:
1) Added a nice error message if a corrupt ProjectVersion.txt is found (does no longer throw an Exception).
2) Added the resizeable columns which I requested in Issue #21.

Great tool, keep on going!

unitycoder commented:
thanks, will check it later this week.

Later probably need to add saving/loading column sizes also to keep users own settings.

geo-at-github commented:
Added saving/loading column sizes just now (see latest pull request). Added them as an int[] which the Visual Studio Settings Designer UI does not seem to like very much. I am new to programming Windows Forms Applications. Maybe there is a better solution for VS2017's Settings Designer.

Also the int[] may cause trouble in the future if you decide to enable reordering the colums (left to right). The settings will just assign the width according to an index 0-3 and don't care about which column it really is. You would have to call "save column widths" again whenever the column order changes. But it's just hypothetical at the moment, so I wouldn't care too much.
